qt/python/mantidqt/project/workspacesaver.py

```diff
@@ -0,0 +1,64 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+#
+
+from mantid.api import AnalysisDataService as ADS, IMDEventWorkspace
+from mantid.dataobjects import MDHistoWorkspace
+from mantid.simpleapi import SaveMD, SaveNexusProcessed
```

Comment:
Similar to above. Can we move this line inside the `save_workspaces` method? It requires kicking up the mantid framework and we don't want that done to early.

Reply:
Done :+1: